tests/__init__.py

```diff
@@ -5,6 +5,3 @@
 
 # All APIs call in tests should go to a local test server.
 os.environ["KAGGLE_API_ENDPOINT"] = "http://localhost:7777"
```

Comment:
Keeping this call to os.environ to ensure we don't accidentally call kaggle.com (this value should never be changed except via patch)